theano/sandbox/cuda/opt.py

```diff
@@ -1896,7 +1891,7 @@ def local_gpu_downsample_factor_max(node):
             return
         x, = node.inputs
         if (x.owner and isinstance(x.owner.op, HostFromGpu)):
-            gpu_ds = GpuDownsampleFactorMax(**node.op._props_dict())
+            gpu_ds = GpuDownsampleFactorMax(node.op.ds, node.op.ignore_border)
```

Comment:
`GpuDownsampleFactorMax` needs to have `__props__`. After that, passing `_props_dict()` would fix I suppose ?